fmriprep/workflows/bold/outputs.py

```diff
@@ -216,13 +216,22 @@ def init_func_derivatives_wf(
                                          ('anat2bold_xfm', 'in_file')]),
     ])
 
-    if nonstd_spaces.intersection(('func', 'run', 'bold', 'boldref', 'sbref')):
+    bold_output = nonstd_spaces.intersection(('func', 'run', 'bold', 'boldref', 'sbref'))
+    if bold_output:
         ds_bold_native = pe.Node(
             DerivativesDataSink(
                 base_directory=output_dir, desc='preproc', compress=True, SkullStripped=masked,
                 TaskName=metadata.get('TaskName'), **timing_parameters),
             name='ds_bold_native', run_without_submitting=True,
             mem_gb=DEFAULT_MEMORY_MIN_GB)
+
+        workflow.connect([
+            (inputnode, ds_bold_native, [('source_file', 'source_file'),
+                                         ('bold_native', 'in_file')]),
+        ])
+
```

Comment:
So we don't get burned by the formatter
```suggestion
        # fmt: off
        workflow.connect([
            (inputnode, ds_bold_native, [('source_file', 'source_file'),
                                         ('bold_native', 'in_file')]),
        ])
        # fmt:on
```